backend/data/src/main/resources/I18n_en.properties

```
@@ -80,4 +80,26 @@ slack.common.message.global-subscription-error=You can't set global subscription
 slack.common.message.modification-info=*%s* has changed the %s at %s
 slack.common.message.feature-not-available=This feature is not available for your plan. Please upgrade your plan to use this feature.
 slack.common.message.bot-not-in-channel=Bot is not in the channel. Please invite the bot to the channel using `/invite @Tolgee` and try again.
-slack.help.message.advanced-subscribe-events-example=`/tolgee subscribe 1 fr --on new_key` - subscribe to new key events in project 1 for French translations
\ No newline at end of file
+slack.help.message.advanced-subscribe-events-example=`/tolgee subscribe 1 fr --on new_key` - subscribe to new key events in project 1 for French translations
+
+notifications.email.subject.TASK_ASSIGNED=New task assignment
```

This is out of convention. It should be 
```suggestion
notifications.email.subject.TASK_ASSIGNED=New task has been assigned to you
```